CSV_Modifier_Client/Controllers/AwsBucketReader.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Amazon;
+using Amazon.S3;
+using Amazon.S3.Model;
+using CSV_Modifier_Client.Models;
+using CSV_Modifier_Client.Services;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSV_Modifier_Client.Controllers
+{
+    public class AwsBucketReader : Controller
+    {
+        private readonly string bucketName = "csv-files-s3-bucket";
+
+        //fetching this file (object) from s3 bucket
+        private readonly string objectKey = "CommaSeparatedFile.csv";
+        private readonly AwsSecretsService _awsSecretsService;
+
+        public AwsBucketReader(AwsSecretsService awsSecretsService)
+        {
+            _awsSecretsService = awsSecretsService;
+        }
+        public async Task<IActionResult> Index()
+        {
+            //super user manager => superman
+            string superman_access_key = await _awsSecretsService.GetAwsSecret("superman_access_key");
```

Comment:
use "var" and naming convention should be like "supermanAccessKey"

Reply:
done